src/UserStories/UserStories.css

```diff
@@ -139,64 +146,49 @@ html {
   border-radius: 22px;
   height: 44px;
   padding-left: 30px;
-  margin-bottom: 25px;
+  margin-bottom: 10px;
   box-sizing: border-box;
   align-self: flex-end;
   background: none;
   text-overflow: ellipsis;
-
   white-space: nowrap;
   overflow: hidden;
 }
 ::-webkit-input-placeholder {
   color: white;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica,
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica,
     Arial, sans-serif;
   max-width: 110px;
 }
 .direct-story-icon {
   align-self: flex-end;
-  margin-bottom: 35px;
+  margin-bottom: 20px;
   padding: 0 5px 0 15px;
 }
 
 #user-avatar-small {
   position: absolute;
-  top: 250%;
+  top: 280%;
   right: 45%;
-  transform: scale(2);
+  transform: scale(2.5);
 }
-.username-small {
+.story-username.is-small {
   position: absolute;
-  top: 300%;
+  top: 420%;
+  transform: scale(2.5);
   right: 43%;
   display: block;
   line-height: 16px;
-  max-width: 60px;
+  max-width: 75px;
   overflow: hidden;
   text-align: center;
   text-overflow: ellipsis;
   white-space: nowrap;
 }
-.secondary-story-post-time {
+
+.story-post-time.is-small {
   position: absolute;
-  top: 330%;
+  top: 470%;
   right: 50%;
-}
-.secondary-story-header {
-  box-sizing: border-box;
-  color: white;
-  display: grid;
-  grid-template-columns: min-content min-content min-content 1fr;
-
-  background: linear-gradient(
-    180deg,
-    rgba(38, 38, 38, 0.8) 0%,
-    rgba(38, 38, 38, 0) 100%
-  );
-  padding: 10px;
-  height: 100px;
-  margin-bottom: -100px;
-  z-index: 2;
-  position: relative;
-}
+  transform: scale(2);
```

Comment:
Same issue as above. These shouldn't be absolute. Absolute is almost always a can of worms.